lambda/update/parse/changes.go

```diff
@@ -123,6 +142,43 @@ func (p *Parser) Field(key string, v any, opts ...Option) *Parser {
 	return p
 }
 
+func oldEqualsExisting(old any, existing any) bool {
+	if v, ok := existing.(*time.Time); ok {
+		if old == nil {
+			return v.IsZero()
+		}
+
+		RFC3339local := "2006-01-02T15:04:05Z"
+		return old.(string) == v.Format(RFC3339local)
+	}
+
+	if v, ok := existing.(*shared.Lang); ok {
+		if old == nil {
+			return *v == shared.LangNotSet
```

Comment:
maybe these could be inlined to `shared.Lang/Channel("")` as we don't really want them to be referenced anywhere else